msteams-platform/apps-in-teams-meetings/teams-live-share-capabilities.md

```diff
@@ -212,42 +203,45 @@ function onClickAddToPlaylist(video: IVideo) {
-| [EphemeralPresence](/javascript/api/@microsoft/live-share/ephemeralpresence) | See which users are online, set custom properties for each user, and broadcast changes to their presence.                               |
-| [EphemeralEvent](/javascript/api/@microsoft/live-share/ephemeralevent)       | Broadcast individual events with any custom data attributes in the payload.                                                             |
-| [EphemeralState](/javascript/api/@microsoft/live-share/ephemeralstate)       | Similar to SharedMap, a distributed key-value store that allows for restricted state changes based on role, for example, the presenter. |
+| Live Object                                                        | Description                                                                                                                             |
+| ------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------- |
+| [LivePresence](/javascript/api/@microsoft/live-share/livepresence) | See which users are online, set custom properties for each user, and broadcast changes to their presence.                               |
+| [LiveEvent](/javascript/api/@microsoft/live-share/liveevent)       | Broadcast individual events with any custom data attributes in the payload.                                                             |
+| [LiveState](/javascript/api/@microsoft/live-share/livestate)       | Similar to SharedMap, a distributed key-value store that allows for restricted state changes based on role, for example, the presenter. |
+| [LiveTimer](/javascript/api/@microsoft/live-share/livetimer)       | Synchronize a countdown timer for a given interval.                                                                                     |
 
-### EphemeralPresence example
+### LivePresence example
 
-The `EphemeralPresence` class makes tracking who is in the session easier than ever. When calling the `.initialize()` or `.updatePresence()` methods, you can assign custom metadata for that user, such as name or profile picture. By listening to `presenceChanged` events, each client receives the latest `EphemeralPresenceUser` object, collapsing all presence updates into a single record for each unique `userId`.
+:::image type="content" source="../assets/images/teams-live-share/live-share-presence.png" alt-text="Teams Live Share presence":::
+
+The `LivePresence` class makes tracking who is in the session easier than ever. When calling the `.initialize()` or `.updatePresence()` methods, you can assign custom metadata for that user, such as name or profile picture. By listening to `presenceChanged` events, each client receives the latest `LivePresenceUser` object, collapsing all presence updates into a single record for each unique `userId`.
```

Comment:
Here and in the Live Canvas we talk about assigning custom metadata for user's names and profile pictures. Should we call out somewhere (maybe in the FAQ) that these values when passed in the event body can not be trusted and someone with the same role could send an event with another person's name and profile pic

Reply:
I will update this in a separate PR. This needs to be posted today ahead of Ignite, but a good callout!